specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2021-12-01/bms.json

```diff
@@ -12434,6 +12478,46 @@
           }
         }
       }
+    },
+    "TieringPolicy": {
+      "description": "Tiering Policy.",
+      "type": "object",
+      "properties": {
+        "tieringMode": {
+          "description": "Tiering Mode to control automatic tiering of recovery points",
+          "enum": [
+            "Invalid",
```

Comment:
>Invalid [](http://example.com/codeflow?start=13&length=7)

When would a user specify this?

Reply:
It's the default value of the enum and not intended to be explicitly specified by user. Any string value specified by user that is not part of the enum gets translated into "invalid" on service side.